test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWithSmartRouterHttpScalingIntegrationTest.java

```diff
@@ -47,7 +48,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category({OperatorNotSupported.class}) // Because DroolsServerFilterClasses not supported yet, Operator skipped as there is discrepancy between deployment methods and Kie server connection, should be unified for 7.4  
+@Category({OperatorNotSupported.class, MonitoringK8sFs.class}) // Because DroolsServerFilterClasses not supported yet, Operator skipped as there is discrepancy between deployment methods and Kie server connection, should be unified for 7.4  
```

Comment:
Is this correct? This integration test will be executed in Templates only and the MonitoringK8sFs is only for Operator.